crates/proto/src/dnssec/tbs.rs

```diff
@@ -51,66 +49,19 @@ impl TBS {
     ///
     /// * `name` - labels of the record to sign
     /// * `dns_class` - DNSClass of the RRSet, i.e. IN
-    /// * `sig` - SIG or RRSIG record, which was produced from the RRSet
+    /// * `input` - `SigInput` data used to create the signature`
```

Comment:
nit: unbalanced backticks
```suggestion
    /// * `input` - `SigInput` data used to create the `signature`
```